CompilerSource/general/bettersystem.cpp

```diff
@@ -346,9 +346,16 @@ void myReplace(std::string& str, const std::string& oldStr, const std::string& n
 
       int result = -1;
       pid_t fk = fork();
+      setpgid(0,0); // new process group
 
       if (!fk)
       {
+        // Redirect STDIN
```

Comment:
Later on the IDE could provide a virtual terminal for the game, and that could be used as in here, but that doesn't matter if we change ENIGMA to have the IDE launch the game. Regardless, gradlew still needed an STDIN to read from that wasn't connected to the terminal.